methods/server.js

```diff
@@ -1,7 +1,32 @@
 // import all methods here (export must be a function)
 import doSomething from './method1'
+import Presentations from '../modules/db/Presentations.js'
+const getSVG = Meteor.npmRequire('google-slides-downloader')
 
 // add to combined Meteor.methods, which will have the var name.
 Meteor.methods({
-  doSomething,
+  createPresentation: function (url, id, gid, cb) {
```

Comment:
So, here, we're going to just import the functions from a separate folder which contains all of your methods. We want the methods block to look like

import method1 from './method1file'
import method2 from './method2file'
import method3 from './method3file'

Meteor.methods({
method1,
method2,
method3
})